pages/common/cpio.md

```diff
@@ -0,0 +1,16 @@
+# cpio
+
+> Copies files in and out of archives.
+> Supports following archive formats: binary, old ASCII, new ASCII, crc, HPUX binary, HPUX old ASCII, old tar, and POSIX.1 tar.
+
+- Take a list of file names from standard input and copy them [o]ut to an archive in binary format:
+
+`echo "{{file1}} {{file2}} {{file3}}" | cpio -o > {{archive.cpio}}`
+
+- Copy [o]ut all files and folders in a directory to an archive, in [v]erbose mode:
+
+`find {{path/to/directory}} | cpio -ov > {{archive.cpio}}`
+
+- Copy [i]n all files from an archive, generating [d]irectories where needed, in [v]erbose mode:
```

Comment:
What a perfectly missed opportunity they had there :disappointed: let's see how we can work around this. How about "add files onto an archive" for `-o` and "import files from an archive" for `-i`? Or maybe "p[i]ck files from an archive", since "import" tends to have a specific meaning that's not exactly matched here.